### DISCUSSION TITLE
reset keyboard shortcuts to default settings

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#825] [Ideas]
### What feature would you like to see?

when you mess around the settings for the shortcuts you cannot reset it to default settings

i imagine just a button on the top right of the page to reset to default settings, like in the home settings page, where you have a "Restore default" button

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/482_

**Comment by luigigorlero** [comment]
It would be useful to have a “Restore” icon next to each shortcut input field, too, in case I made a full configuration I’m happy with and wish to restore just a single one to its default settings.

**Reply by TiermoInvin** [reply]
Yeah its really sad that in order to restore a single one, you have to reset all. Doubly more confusing when you don't know the original.

**Comment by robotkutya** [comment]
Yeah, this is very annoying :(

The UI automatically picks up changes, and then puff, they are gone forever. It's also quite annoying that mac keyboards don't have an "Alt" key, the "option" key is registered as "Meta"

**Comment by yoonthegoon** [comment]
A number of shortcuts are interfering with default Mac hotkeys.
Currently cannot quit with `Cmd + Q`, cannot open settings with `Cmd + ,`, attempting to find in page with `Cmd + F` enters full screen, etc.
